core/domain/services/admin/events/editor/EventEditorGraphQLData.php

```diff
@@ -89,7 +95,8 @@ public function __construct(
         EventEntityRelations $relations,
         EventManagerData $managers,
         NewEventDefaultEntities $default_entities,
-        TicketMeta $ticket_meta
+        TicketMeta $ticket_meta,
+        FormBuilder $form_builder
```

Comment:
not your problem or your fault, but this constructor is getting way too crowded 
10 parameters ?!?!?! yikes 😬